server/pbench/bin/index-pbench

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3 -u
```

Comment:
So this forces unbuffered mode, right?  But is that necessary for the final commit?